package.json

```diff
@@ -11,7 +11,9 @@
     "clean": "lerna clean && lerna run clean",
     "format": "prettier --write \"packages/**/*.{ts,js,tsx,jsx,json}\"",
     "lint": "eslint packages/ --ext .js,.ts,.jsx,.tsx",
-    "test:lhci": "lhci autorun --collect.url=/index.html"
+    "test:lhci": "lhci autorun --collect.url=/index.html",
+    "bootstrap": "lerna bootstrap",
+    "master-publish": "lerna version minor --yes && lerna publish from-git --yes"
```

Comment:
why `master-publish` and not only `publish` ?

Reply:
Because it's a script that can only be used in the master branch, buut I think it feels weird, you have a point. Gonna change to `publish`.